src/pages/ProductList/ImageSlide/ImageSlide.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import SlickDot from './SlickDot/SlickDot';
+import Slide from './Slide/Slide';
+import { SLIDE_DATA } from './SlideData';
+import './imageSlide.scss';
+
+const ImageSlide = () => {
+  const [countSlide, setCountSlide] = useState(0);
+
+  const changeSlide = slideId => {
+    setCountSlide(slideId);
+  };
+
+  return (
+    <div className="imageSlide">
+      <div className="slideList">
+        {SLIDE_DATA.filter(el => el.id === countSlide).map(el => (
+          <Slide key={el.id} slideId={el.id} imgUrl={el.imageUrl} />
+        ))}
+      </div>
+      <ul className="slickDotWrapper">
+        {SLIDE_DATA.map(el => (
+          <SlickDot
+            key={el.id}
+            currentSlideId={countSlide}
+            slideId={el.id}
+            changeSlide={changeSlide}
```

Comment:
```suggestion
            changeSlide={() => changeSlide(slide.id)}
```
두개의 props를 받아서 하나의 함수를 동작시키기 위해서 하는 것 같은데 애초에 하나의 함수만 만들어서 props로 전달하는 방식으로 진행하는게 좋아보이네요!

Reply:
changeSlide는 props로 바로 내려줄수 있도록 했습니다.
다만 slideId같은 경우는 currentSlideId와 비교하여 현재 선택된 버튼의 스타일 변경을 위해 props로 내려주었습니다.
혹시 slideId를 따로 내려주지 않는 다면 다른 방식으로 css 변경을 할 수 있는 방법이 있는지 여쭤봐도 될까요?

Reply:
이부분도 isSelected={slideId === currentSlideId} 같이 하나의 props로 전달하는게 컴포넌트가 좀 더 알아보기 쉬워보이네요!!
위에서 미리 비교해서 필요한 결과값만 내려주는 식으로 진행하시면 될 것 같습니다!

마이너한 사항이라서 일단 머지하겠습니다! 다음에 반영해주세요

Reply:
네 참고하겠습니다!